service/src/main/resources/static/locations.yml

```diff
@@ -335,14 +353,17 @@ locations:
       description: Asia Southeast 1 (GCP)
       cloudRegion: asia-southeast1
       cloudPlatform: gcp
-    - name: gcp.asia-northeast2
-      description: Asia Northeast 2 (GCP)
-      cloudRegion: asia-northeast2
-      cloudPlatform: gcp
     - name: azure.southeastasia
       cloudRegion: southeastasia
       cloudPlatform: azure
       description: Southeast Asia (Azure)
+  - name: indonesia
```

Comment:
I think this follows our pattern since there's only one in the country. We could make it more granular and go with Jakarta instead.